src/main/java/org/zaproxy/zap/extension/hud/HudParam.java

```diff
@@ -328,6 +376,16 @@ public void setUiOption(String key, String value) {
         }
     }
 
+    public void clearNewChangelog() {
+        this.newChangelog = false;
+        getConfig().setProperty(PARAM_NEW_CHANGELOG, newChangelog);
+        try {
+            this.getConfig().save();
+        } catch (ConfigurationException e) {
+            log.error(e.getMessage(), e);
+        }
```

Comment:
Extract a saveConfig method to use in these clear and update functions?

Reply:
too obvious ;)